src/test/java/com/treasuredata/client/TestTDClient.java

```diff
@@ -150,17 +150,17 @@ public void setUp()
     public void tearDown()
             throws Exception
     {
-        for (String name : savedQueries) {
-            try {
-                client.deleteSavedQuery(name);
-            }
-            catch (Exception e) {
-                logger.error("Failed to delete query: {}", name, e);
+        try (TDClient client1 = client;
```

Comment:
I'm not sure whether it's necessary to use try-with-resource statement in this case.

The resources are declared in another method. For that case, I assume that it's not necessary.

Reply:
I think try-with-resource is better since it ensures that `server` is closed even when `client.deleteSavedQuery` throws exception.

Reply:
sure.